ui/components/req-filter-ui.jsx

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios';
+import React from 'react';
+import { Link } from 'react-router';
+
+import { apiUrl } from '../globals';
+
+function FilterItem({ keywordIds, keyword, query }) {
```

Comment:
I don't want to be nitpicky, and I don't know what the standard is wrt naming React components, but might we want to consider simplifying naming here by changing `FilterItem` -> `Filter` and `ReqFilterUI` to something like `FilterList`. To my mind this feels clearer, but I could be alone in this. Certainly not something to hold up this PR. I know it glosses over the fact that we will have other filters in the future, and, to my mind, we might do something that looks like https://facebook.github.io/react/docs/composition-vs-inheritance.html#specialization where we compose elements that are special versions of `Filter` and `FilterList` or whatever they are called, and create a more generic implementation of these components when the time comes.